languages.toml

```
@@ -2336,6 +2338,21 @@ language-servers = [ "jsonnet-language-server" ]
 name = "jsonnet"
 source = { git = "https://github.com/sourcegraph/tree-sitter-jsonnet", rev = "0475a5017ad7dc84845d1d33187f2321abcb261d" }
 
+[[language]]
+name = "ada"
+scope = "source.ada"
+injection-regex = "ada"
+file-types = ["adb", "ads", "gpr"]
+roots = ["alire.toml", "*.gpr"]
```

Currently `roots` doesn't support globs, just full filenames

```suggestion
roots = ["alire.toml"]
```

---

That's bad, but okay, will adjust it accordingly. For now it will work only with alire-based projects then

---

Additional question - only `*.ext` don't work or it also applies to `.ext` ?

---

Currently extensions are not allowed either, just full file names. For some languages a glob pattern would make sense, like if the build system expects `<projectname>.<extension>`. I'll make an issue for that when I get a chance